bot/commands/config.py

```
@@ -0,0 +1,455 @@
+
+        await interaction.response.defer(ephemeral=True)
+
+        try:
+            if key not in CONFIGURABLE_OPTIONS:
+                raise ConfigValidationError(self.translations["config_error_invalid_key"])
+
+            # Handle special case where value is "null", "none", or "XX:XX" for FIXED_UPDATE_TIME
+            if value and value.lower() in ("null", "none"):
+                value = None
+            elif key == "FIXED_UPDATE_TIME" and value and value.upper() == "XX:XX":
+                value = None
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)